src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -42,25 +42,20 @@
     * Create an {@service.clientName} with additional helpers for common
     * tasks.
     *
-    * @@param {Object=} opts - The optional parameters.
-    * @@param {String=} opts.servicePath
-    *   The domain name of the API remote host.
-    * @@param {number=} opts.port
-    *   The port on which to connect to the remote host.
-    * @@param {grpc.ClientCredentials=} opts.sslCreds
-    *   A ClientCredentials for use with an SSL-enabled channel.
-    * @@param {Object=} opts.clientConfig
-    *   The customized config to build the call settings. See
-    *   {@@link gax.constructSettings} for the format.
+    {@comments(@service.doc.lines)}
+    *
+    * @@param {Object=} options - The optional parameters.
+    * @@param {number=} options.port - The port on which to connect to the remote host.
+    * @@param {String=} options.servicePath - The domain name of the API remote host.
     */
-   function {@service.localName}(opts) {
+   function {@service.localName}(options) {
      // Define the header options.
-     opts = opts || {};
-     opts.libName = 'gccl';
-     opts.libVersion = VERSION;
+     options = options || {};
```

Comment:
Do you think we should clone the user's input object before assigning properties to it?